mis_builder/models/mis_report.py

```diff
@@ -334,7 +352,9 @@ def name_search(self, name='', args=None, operator='ilike', limit=100):
             ]
```

Comment:
(('kpi_id.name', '=', kpi_name), ('subkpi_id.name', operator, subkpi_name)) or (kpid_id.description operator name) or (subkpi.description operator name)

Reply:
I changed it. I also always search on kpi name and kpi description, event if there is a dot in the name.